gatsby-node.js

```diff
@@ -237,8 +249,9 @@ async function runIndexQueries(
 
     /* Add changed / new objects */
     const chunkJobs = chunks.map(async function (chunked) {
-      const { taskID } = await indexToUse.addObjects(chunked);
-      return indexToUse.waitTask(taskID);
+      await indexToUse.saveObjects(chunked, {
+        autoGenerateObjectIDIfNotExist: true,
```

Comment:
objectID is required, so we can use saveObjects without autoGenerateObjectIDIfNotExist